notebooks/00_quick_start/run_notebook_on_aml.ipynb

```diff
@@ -0,0 +1,2380 @@
+{
```

Comment:
<p>one question, what you are doing is creating a new VM, executing the notebook there and then turn off the machine? or are you attaching a machine that is already created?</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/681/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I used on-demand compute target called amlcompute. The computing resource are recycled by default. See https://docs.microsoft.com/en-us/azure/machine-learning/service/how-to-set-up-training-targets#amlcompute